docs/ballerina-by-example/examples/vector/vector.bal

```diff
@@ -0,0 +1,28 @@
+import ballerina.collections;
+
+function main (string[] args) {
+    // Initialize the vector with a new array.
+    collections:Vector vec = {vec:[]};
+
+    // Add elements to the vector. Elements of any type can be added to a vector.
+    vec.add("Ballerina");
+    vec.add("Vector");
+
+    // size() function returns the size of the vector.
+    println("No. of elements in the vector: " + vec.size());
+
+    // The insert() function can be used to insert elements at the user specified index.
+    // An IndexOutOfRangeError will be returned if the specified index does not lie within 0 and the size of the vector.
```

Comment:
It would be great if we clarify this further saying that the last index +1 element also eligible for the insertion

Reply:
Ack. Will rephrase this.